modules/data.remote/R/call_MODIS.R

```diff
@@ -13,7 +13,10 @@
 ##' @param band   string value for which measurement to extract
 ##' @param band_qc string value for which quality control band, or use "NA" if you do not know or do not need QC information (optional)
 ##' @param band_sd string value for which standard deviation band, or use "NA" if you do not know or do not need StdDev information (optional)
+##' @param siteID numeric value of BETY site id value to use for output file name. Default is NULL
```

Comment:
This is just optional correct?  We would want this tool to be useful even if you just provide a lat long/ . Can we instead allow for a BETY site ID OR a custom label to be attached to the output?

Reply:
Yea. This is really useful naming output files in a way that is easily identifiable. It will be really helpful in the future when we start adding in more sites. We won't have to re-download sites because I'll have it check for the site in the file name before running call_MODIS

Reply:
if there is no site name, it uses the lat/lon... which just makes the output really long